cmake/ThrustCudaConfig.cmake

```diff
@@ -1,6 +1,6 @@
 enable_language(CUDA)
 
-set(THRUST_KNOWN_COMPUTE_ARCHS 35 37 50 52 53 60 61 62 70 72 75 80 86)
+set(THRUST_KNOWN_COMPUTE_ARCHS 35 37 50 52 53 60 61 62 70 72 75 80 86 90)
```

Comment:
fwiw, we could drop 35/37 now as those are formally no longer supported.

Reply:
Let's revisit this in the future -- it may be useful to keep the options around for testing against older CTK versions.

Also, we'll be stripping all of this special arch handling out when we remove the nvc++ hacks, and replacing it with the standard `CUDA_ARCHITECTURES` cmake properties.